lib/ansible/modules/clustering/pacemaker_docker.py

```diff
@@ -0,0 +1,300 @@
+#!/usr/bin/python
+# coding: utf-8 -*-
+
+# Copyright (c), Chafik Belhaoues  <chafik.bel@gmail.com>, 2017
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+
+DOCUMENTATION = '''
+---
+module: pacemaker_docker
+short_description: Manage docker resources in a pacemaker cluster
+version_added: "2.5"
+author: "Chafik Belhaoues (@africanzoe)"
+description:
+   - This module can create|delete|manage|unmanage a docker resource within a
+     pacemaker cluster using the pacemaker "pcs" cli.
+options:
+    state:
+      description:
+        - Indicate desired state of the resource
+      choices: ['created', 'deleted', 'managed', 'unamanged']
```

Comment:
typo of unmanged ?

I think state: present, absent

managed: bool
may be clearer and more "Ansible Like"

Reply:
You are right about the typo of "unmanged" in the description (not in the code).
Concerning the state: present|absent, actually I thought that it will be more accurate to use Pacemaker terms to be more specific but I agree with you as Ansible manages states, it's more clearer and more "Ansible like" to use those words instead.
I'll make the necessary changes and push again.